.travis.yml

```diff
@@ -42,27 +43,22 @@ services:
 env:
   - REQUIREMENTS=lowest BROKER_URL=redis://localhost:6379/0
   - REQUIREMENTS=lowest BROKER_URL=amqp://localhost:5672//
-  - REQUIREMENTS=release BROKER_URL=redis://localhost:6379/0
-  - REQUIREMENTS=release BROKER_URL=amqp://localhost:5672//
+  - REQUIREMENTS=release BROKER_URL=redis://localhost:6379/0 DEPLOY=true
+  - REQUIREMENTS=release BROKER_URL=amqp://localhost:5672// DEPLOY=true
```

Comment:
Remove one DEPLOY flag (I think any is fine), otherwise this will attempt to push the package to PyPI twice.